v2/cmd/config/secrets.go

```diff
@@ -51,7 +58,7 @@ func (s *Secrets) getPGPKeyRing() (*pgp.KeyRing, error) {
 
 	for name, key := range s.Keys {
 		err := pgp.AddKeyToKeyRingFromReader(newKeyRing, name, bytes.NewReader([]byte(key)))
-		if nil != err {
+		if err != nil {
```

Comment:
Putting the value before the variable is an established pattern in this repo (to prevent accidental assignments) so I think we should leave it

Reply:
SGTM! I don't think it's idiomatic Go, but the [staticcheck](https://staticcheck.io/docs/checks#ST1017) rule that enforces this isn't enabled so 🤷 .

> ST1017 – Don't use Yoda conditions
Yoda conditions are conditions of the kind `if 42 == x`, where the literal is on the left side of the comparison. These are a common idiom in languages in which assignment is an expression, to avoid bugs of the kind `if (x = 42)`. In Go, which doesn't allow for this kind of bug, we prefer the more idiomatic `if x == 42`.

Accepted, this is.